docs/content/latest/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -15,15 +15,149 @@ isTocNested: true
+
+### YCQL Advanced
+
+| **Graph**                     | **Description**                                              | **Alert Guidance**                                           | **Example**                      |
+| ----------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | -------------------------------- |
+| YCQL Latency Breakdown        | The average time (in milliseconds) spent by the CQL API parsing and executing operations. | Information pending                                          | ![img](/images/yp/metrics15.png) |
+| YBClient Ops Local vs Remote  | The count of local and remote read and write requests.<br>Local requests are executed on the same node that has received the request.<br>Remote requests are re-routed internally to a different node for executing the operation. | If an application is using a Yugabyte driver that supports local query routing optimization and prepared statements, the expected value for this is close to 100% local for local reads and writes.<br>If using the Cassandra driver or not using prepared statements, expect to see a relatively even split (for example, ~33% local and ~66% remote for a 3-node cluster). | ![img](/images/yp/metrics16.png) |
+| YBClient Latency              | Latency of local and remote read and write requests.<br>Refer to the YBClient Ops Local vs Remote description regarding local and remote requests. | Information pending                                          | ![img](/images/yp/metrics17.png) |
+| Reactor Delays                | Information pending                                          | This value should be close to zero. If it is increasing or stays high, you should treat it as an indicator that the system is being overloaded or that queues are backing up. If this is the case, you should investigate throughput and queue size and latency metrics for tuning guidance. | ![img](/images/yp/metrics18.png) |
+| RPC Queue Size                | The number of RPCs in the service queue.                     | The queue size is an indicator of incoming traffic and throughput. Typically, it either shows very low values, which means that Yugabyte Platform is processing requests at a healthy rate, or a flat line at the maximum-configured queue size, which means that Yugabyte Platform has filled the queues and cannot process fast enough. | ![img](/images/yp/metrics19.png) |
+| Response Size (bytes)         | Information pending                                          | Information pending                                          | ![img](/images/yp/metrics20.png) |
+| Transaction                   | Information pending                                          | Information pending                                          | ![img](/images/yp/metrics21.png) |
```

Comment:
Added the following to the Description column:
_The number of transactions._

Added the following to the Alert Guidance column:
_This value depends on the application or activity. Since transactions can have batched statements, it is not possible to provide a specific guidance for this metric._